liota/dccs/iotcc.py

```diff
@@ -422,6 +432,7 @@ def store_device_info(self, uuid, name, dev_type, prop_dict):
 
     def write_entity_file(self, prop_dict, res_uuid):
         file_path = self.entity_file_path + '/' + res_uuid + '.json'
+        prop_dict.update({"Entity_Timestamp": datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")})
```

Comment:
Although it warns like that, it turned out that is not a problem for all the gateways we have.
I tried it on all the gateways in our lab (of course it works in my mac os, linux ubuntu VMs, etc.), and all are successfully installed.